### PR TITLE
Remove Contacts/People/modules.html

### PR DESCRIPTION
**Pull request by ChrisJefferson**
These removes Contacts/People/modules.html, which is both out of date and not greatly helpful. Had to remove some text which linked to it too.

### CODE REVIEW

File: Contacts/People/people.html [hunk 1]
```diff
@@ -24,40 +24,6 @@ <h3>
...
-  to GAP that are not definable as modules, but may be even
-  considered as the backbone of the system. Of these, in the first place
-  Martin Schönert should be mentioned who has been the system architect
-  of GAP from its beginning until 1996. In that period
-  (until 1997) also Frank Celler has made large contributions to the system.
-  Also from the beginning of GAP and until now, Thomas
-  Breuer has made large contributions not only to his special area of
-  work, namely representation theory in GAP. Steve Linton,
-  Alexander Hulpke, and Frank Lübeck have taken over a good deal of
-  Martin's rôle since 1997.
```
Comment (fingolfin) [inline Contacts/People/people.html]: So the one thing to mention is that this kind of thing could have been useful for grant applications. It still might be, if it was updated to reflect the current reality... But perhaps there are other better ways to do that these days, e.g. using the GitHub "insights" features?

**Comment by fingolfin** [comment]
It would be good to know what affected people think about this removal, e.g. @hulpke @ThomasBreuer @beick

**Comment by ThomasBreuer** [comment]
Thanks for the hint about this pull request.

I find it surprising that this text is still present, taking into account that the corresponding list has been removed from the GAP manuals in April 2012, see Commit v4.5.3.
Note that the file `tut/authors.xml` is still there, it is just not included in `ref/main.xml` and `tut/main.xml`. Perhaps it is better to remove `tut/authors.xml`.

Concerning outdated statements in the GAP webpages, there are certainly more. The most prominent one is probably the one about that "Jenks prize" on the main page, which dates back to 2008.

**Comment by hulpke** [comment]
I'm fine with removing this text as in this PR, but  I think the Jenks prize link should stay prominent, even if it is >10 years old.

**Comment by beick** [comment]
Dear all,

>I'm fine with removing this text as in this PR, but  I think the Jenks prize 
>link should stay prominent, even if it is >10 years old.

I agree to that.

Best wishes,

Bettina

**Comment by ThomasBreuer** [comment]
Concerning the Jenks prize link, I remember an e-mail message by @stevelinton from July 2008 where he proposed that the statement
> should go on the main page prominently as news and then shrink in a few weeks back to a link in a corner somewhere.

**Comment by Stefan-Kohl** [comment]
Has the concept of modules and module maintainers been abandoned? -- And if yes, when has this happened, and are there any records on this?

**Comment by fingolfin** [comment]
@Stefan-Kohl yes, it has been abandoned. I am not aware of a "formal decision" (there was a discussion about this on the GAP devel list in May 2012; I had forgotten about this, but Thomas' comment above made me search the archive). However, it is a matter of the actual facts: Since I've been involved in GAP development (around 2011), I've never observed these modules playing any role or being considered in the course of anything related to the development of GAP. In fact I bet everybody who first submitted a patch to GAP in the past 5 years or so is not even aware these modules and module maintainers exist.

So, since they have no practical meaning, and since they were removed from the GAP manual already in 2012 (with the argument back then that they play no actual role), this PR just updates the website to reflect reality.

**Pull request merged**